Tests/InstallationTests/CarthageInstallation/fastlane/Fastfile

```diff
@@ -16,6 +16,14 @@ lane :installation_tests do
     # doesn't try to build the other installation tests
     sh "carthage", "update", "--no-build"
     sh "rm", "-rf", "Carthage/Checkouts/purchases-root/Tests/InstallationTests/"
+
+    # Carthage builds all schemes including ones we don't need, so let's nuke them before proceeding
+    schemes_directory = "Carthage/Checkouts/purchases-root/RevenueCat.xcodeproj/xcshareddata/xcschemes"
+    Dir.glob("#{schemes_directory}/*.xcscheme").each do |file_path|
+      next if File.basename(file_path) == "RevenueCat.xcscheme"
```

Comment:
does this mean that we're skipping the test for RevenueCatUI?

Reply:
RevenueCatUI scheme is only defined in the Package file, which is ignored by Carthage.

Reply:
Oh wait that might be wrong, it's there but not shown? Anyway, this is the directory it searches, so it shouldn't affect RevenueCatUI.
```
jamesborthwick@Jamess-MacBook-Pro-2 xcschemes % ls -l 
total 48
-rw-r--r--@ 1 jamesborthwick  staff  4978 13 Apr 14:02 BackendIntegrationTests.xcscheme
-rw-r--r--@ 1 jamesborthwick  staff  3591 13 Apr 14:02 ReceiptParser.xcscheme
-rw-r--r--@ 1 jamesborthwick  staff  7220 14 Jun 14:47 RevenueCat.xcscheme
-rw-r--r--@ 1 jamesborthwick  staff  2609 20 Jun 10:21 RevenueCatUIDev.xcscheme
```